MerlinAU.sh

```diff
@@ -1983,20 +1999,22 @@ check_memory_and_prompt_reboot()
     fi
 }
 
-##------------------------------------------##
-## Modified by ExtremeFiretop [2024-Jan-06] ##
-##------------------------------------------##
+##----------------------------------------##
+## Modified by Martinski W. [2024-May-31] ##
+##----------------------------------------##
```

Comment:
Approved the changes below to swap to the long version

Reply:
With all the changes to use the long version, do we really need the short version much anymore? I'm assuming it can be removed correct?